env/staging/create_metrics_lambda/lambda/create_metric.js

```diff
@@ -97,3 +101,13 @@ const  writePayload = async (payload, ttl) => {
   
   return dynamodb.putItem(params).promise();
 };
+
+const  saveSample = async (data, key) => {
+  var params = {
+      Bucket : process.env.BUCKET_NAME,
+      Key : key,
+      Body : data
+  }
+
+  return s3.putObject(params).promise(); 
```

Comment:
Just a minor comment - we might want something like this to troubleshoot if it doesn't work as expected:
```js
  try {
    const stored = await s3.upload(params).promise()
    console.log(stored);
  } catch (err) {
    console.log(err)
  }
```

Reply:
This should be caught by the main try catch block since i'm returning the promise.

Reply:
Ah, totally missed that!  Move along, nothing to see here 😅